docs/config/README.md

```diff
@@ -437,6 +438,29 @@ truncation_length = 4
 truncation_symbol = ""
 ```
 
+
+## Git Commit
+
+The `git_commit` module shows the active branch of the repo in your current directory.
```

Comment:
I'd suggest adding a container clarifying that this module is disabled by default, as we do for the `time` module:
https://github.com/starship/starship/blob/ab310e2cf047e6ed57c7519d997d28e70e702fac/docs/config/README.md#L938-L943